src/components/SubmissionConfirmation.js

```diff
@@ -160,10 +160,7 @@ const SubmissionConfirmation = ({statusUrl, onFailure, onConfirmed}) => {
         <Body>
           <FAIcon icon="download" aria-hidden="true" modifiers={['inline']} />
           <Anchor href={reportDownloadUrl} target="_blank" rel="noopener noreferrer">
-            <FormattedMessage
-              description="Download report PDF link title"
-              defaultMessage="Download PDF"
-            />
+            {form.submissionReportDownloadLinkTitle}
```

Comment:
Let's take a conservative and backwards compatible approach here - try to get the value from the API endpoint, but if it's falsy, fall back to the existing `<FormattedMessage>` approach. I don't think this change warrants an SDK major version bump.